diff --git a/openpgm/pgm/thread.c b/openpgm/pgm/thread.c
--- a/openpgm/pgm/thread.c
+++ b/openpgm/pgm/thread.c
@@ -50,7 +50,7 @@
 			if (PGM_UNLIKELY(save_error)) { \
 				char errbuf[1024]; \
 				pgm_error ("file %s: line %d (%s): error '%s' during '%s'", \
-					__FILE__, __LINE__, __PRETTY_FUNCTION__, \
+					__FILE__, __LINE__, __func__, \
 					pgm_strerror_s (errbuf, sizeof (errbuf), save_error), name); \
 			} \
 		} while (0)
@@ -73,7 +73,7 @@
 			const bool save_error = (err); \
 			if (PGM_UNLIKELY(!save_error)) { \
 				pgm_error ("file %s: line %d (%s): error '%s' during '%s'", \
-					__FILE__, __LINE__, __PRETTY_FUNCTION__, \
+					__FILE__, __LINE__, __func__, \
 					pgm_wsastrerror (GetLastError ()), name); \
 			} \
 		} while (0)
